package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "hof-feedback-form",
+  "version": "1.0.0",
+  "description": "A feedback form for users of other HOF services to fill",
+  "main": "index.js",
+  "engines": {
+    "node": ">=18.12.1"
```

Comment:
this can be updated to `>=20.15.0`

Reply:
We should really allow devs to work on 18 LTS at the least, but happy to change

Reply:
The problem is that this will result in a significant difference in the yarn.lock file.
Another reason is that the development environment should be as close as possible to the production environment.

Reply:
AFAIK it will still build the app remotely as defined in the Dockerfile (if defined) but this will stop attempts to build on a lower version. I think its good to be up to date, but means people may need to be careful to move back to 18 when working on existing projects locally.

Reply:
Have changed it to `>=20.15.0`